### DISCUSSION TITLE
I made a commit yesterday and it doesn't show up on the graph.

### DISCUSSION THREAD

**Discussion by ch4rindo** [community/community#37958] [Other Feature Feedback, Questions, & Ideas] [answered]
Commits from private repositories are also displayed.

**Comment by radar-blip** [comment, marked as answer]
Shows you made 3 commits yesterday.

![commits](https://user-images.githubusercontent.com/39196767/199201757-2c03e52b-2aac-465b-9488-4c6d05242271.png)

**Reply by ch4rindo** [reply]
I'm sorry.

**Reply by radar-blip** [reply]
Nothing to be sorry about. Carry on. 😃

**Comment by abhisheksharmayt** [comment]
I am facing the same issue, my commits are not visible in the heatmap even after committing with the correct email in the github.

**Reply by NabilMch** [reply]
Hello friend, If your commits aren’t showing up in the GitHub heatmap, here are a few quick checks:

1. **Correct Email**: Double-check that the email in your Git config matches one in your GitHub account.
   
2. **Branch**: Only commits on the default branch (usually `main` or `master`) are counted.

3. **Private Repo**: If it's a private repo, make sure you’ve enabled “private contributions” in your GitHub profile settings.

4. **Time Delay**: Sometimes it takes up to 24 hours for new commits to appear in the heatmap.

Let me know if this helps!

**Reply by abhisheksharmayt** [reply]
Thank you for your response.
So if I have forked an open-source repo and committing to a new branch, commits will be visible once the changes are merged.
